src/main/java/com/koichi/assignment8/StudentService.java

```diff
@@ -0,0 +1,55 @@
+package com.koichi.assignment8;
+
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class StudentService {
+
+    private final StudentMapper studentMapper;
+
+    public StudentService(StudentMapper studentMapper) {
+        this.studentMapper = studentMapper;
+    }
+
+    public Student findStudent(Integer id) {
+        Optional<Student> findById = this.studentMapper.findById(id);
+        if (findById.isPresent()) {
+            return findById.get();
+        } else {
+            throw new StudentNotFoundException("user not found");
+        }
+    }
+
+    public List<Student> findAllStudents(Integer grade, String startsWith, String birthPlace) {
+        List<Student> getAllStudent = this.studentMapper.findAllStudents();
+        List<Student> getByGrade = this.studentMapper.findByGrade(grade);
+        List<Student> getByStartWith = this.studentMapper.findByName(startsWith);
+        List<Student> getByBirthPlace = this.studentMapper.findByBirthPlace(birthPlace);
+
+        int cont = 0;
+        if (grade != null) {
+            cont = cont + 1;

```

Comment:
Javaの場合はインクリメント演算子が使えますね！
```suggestion
            cont++;
```

Reply:
- インクリメント演算子に変更完了しました。
edb986e57ba5e7b7f3da4ebf349bf1f9cc931ed8